code/obj/item/ai_modules.dm

```diff
@@ -335,6 +336,37 @@ AI MODULES
 		//AI.eyecam.name = lawTarget //not sure if we need?
 
 
+/********************* EXPERIMENTAL LAWS *********************/
+//note: these 'experimental' laws are spawned in via random_item_spawner instead of spawning every round
+//so, any given one of these will only have a chance of spawning
+//[IMPORTANT] The pool for the experimental laws starts on line 810 in file /code/obj/random_spawners.dm. UPDATE THAT TO INCLUDE ANY NEW LAWS SO THEY CAN SPAWN!!!
+//also note: ALL lawNumber of the experimental laws *must* be 13 so that any new ones uploaded overwrite any existing ones
```

Comment:
You can do this via setting `lawNumber` on `/obj/item/aiModule/experimental`, then all of the experimental modules will inherit it (and I guess can override if needed).

Reply:
Gotcha! I'll tweak that real quick

Reply:
Edited and merged. Shouldn't cause any problems.